curriculum/challenges/english/25-front-end-development/lecture-working-with-regular-expressions/6733c5f20cc9584cada942a4.md

```diff
@@ -89,19 +108,25 @@ To define a named capture group, you add a question mark (`?`) followed by the n
 const regex = /free(?<code>co+de)camp.*free\1camp/i;
 ```
 
-Now we can update our backreference in the regular expression to refer to this group. A named back reference starts with a backslash followed by the letter `k` in JavaScript. Then you add the name, again enclosed in less than and greater than signs. Let's take a look at that:
+Now we can update our backreference in the regular expression to refer to this group. A named back reference starts with a backslash followed by the letter `k` in JavaScript. Then you add the name, again enclosed in less than (`<`) and greater than (`>`) signs. Let's take a look at that:
 
 ```js
 const regex = /free(?<code>co+de)camp.*free\k<code>camp/i;
 ```
 
-Now if we check our `test` call, we can see that we still pass:
+Now if we check our `test()` call, we can see that we still pass:
 
-![image](https://cdn.freecodecamp.org/curriculum/lecture-transcripts/what-are-capturing-groups-and-backreferences-and-how-do-they-work-5.png)
+```js
+const regex = /free(?<code>co+de)camp.*free\k<code>camp/i;
+console.log(regex.test("freecooooodecamp is freecooooodecamp")); // true
+```
 
-To use our named capture group in a `replace` call, we'd insert a dollar sign into the string, followed by the name enclosed in less than and greater than signs:
+To use our named capture group in a `replace()` call, we'd insert a dollar sign into the string, followed by the name enclosed in less than and greater than signs:
 
-![image](https://cdn.freecodecamp.org/curriculum/lecture-transcripts/what-are-capturing-groups-and-backreferences-and-how-do-they-work-6.png)
+```js
+const regex = /free(?<code>co+de)camp/i;
+console.log("freecooooodecamp".replace(regex, "paid$<code>camp")); // paidcooooodecamp
+```
 
 Finally, sometimes you want to create a group of characters, but don't need the captured result.
 
```

Comment:
It seems `You can't backreference a non-capturing group.` towards the end of the transcript is not part of the video. I cannot make the suggestion in the right place.